Resources/Prototypes/_LostParadise/Entities/Objects/Misc/identification_cards.yml

```
@@ -27,21 +27,21 @@
-#       color: "#1B67A5"
-#     - state: subdepartment
-#       color: "#1B67A5"
-#     - state: idheadofpersonnel
-#   - type: PresetIdCard
-#     job: LPPAdjutant
+- type: entity
+  parent: IDCardStandard
+  id: LPPAdjutantIDCard
+  name: adjutant ID card
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)